src/raster/rasterband.rs

```diff
@@ -829,13 +844,25 @@ impl PaletteInterpretation {
             _ => unreachable!("GDAL has implemented a new type of `GDALPaletteInterp`"),
         }
     }
+
+    /// Returns the C API int value of this palette interpretation.
+    pub fn c_int(&self) -> GDALPaletteInterp::Type {
+        match self {
+            Self::Gray => GDALPaletteInterp::GPI_Gray,
+            Self::Rgba => GDALPaletteInterp::GPI_RGB,
+            Self::Cmyk => GDALPaletteInterp::GPI_CMYK,
+            Self::Hls => GDALPaletteInterp::GPI_HLS,
+        }
+    }
```

Comment:
Aside: the `from_c_*` and `c_*` duals are a pattern enough in the code base we may want to consider a unify trait?

Reply:
We could, but I'm not sure it would bring us much. We'd have to make the C representation an associated type, which makes those even more awkward to work with.